pkl-cli/src/test/kotlin/org/pkl/cli/CliEvaluatorTest.kt

```diff
@@ -1457,6 +1457,49 @@ result = someLib.x
     assertThat(output).isEqualTo("result = 1\n")
   }
 
+  @Test
+  fun `eval file with non-ASCII name`() {
+    val dir = tempDir.resolve("🤬").createDirectory()
+    val file = writePklFile(dir.resolve("日本語.pkl").toString(), """
+      日本語 = "Japanese language"
+      readDir = read(".").text
+      readDirFile = read("file:$tempDir/🤬").text
+      readOne = read("日本語.pkl").text.split("\n").first
+      readOneFile = read("file:$tempDir/🤬/日本語.pkl").text.split("\n").first
+      readGlob = read*("./日*.pkl").keys
+      readGlobFile = read*("file:$tempDir/**/*.pkl").keys.map((it) -> it.replaceAll("$tempDir", ""))
+      importOne = import("日本語.pkl").readOne
+      importOneFile = import("file:$tempDir/🤬/日本語.pkl").日本語
+      importGlob = import*("./日*.pkl").keys
+      importGlobFile = import*("file:$tempDir/**/*.pkl").keys.map((it) -> it.replaceAll("$tempDir", ""))
```

Comment:
This will fail on Windows thanks to different filepath separators.

Better to do:

```
val tempDirUri = tempDir.toUri()
```

And use that in the code snippet instead, e.g.

```
read("$tempDirUri/日本語.pkl")
```

Reply:
Ah good catch. I think I've got this sorted, but I have no way to test on windows to be sure.